multiply_ui/server/handlers.py

```diff
@@ -85,6 +85,15 @@ def get(self):
         json.dump(parameters, self)
 
 
+# noinspection PyAbstractClass
+class GetRequestHandler(ServiceRequestHandler):
+    def post(self):
+        self.set_header('Content-Type', 'application/json')
+        parameters = self.get_body_as_json_object()
+        request = controller.get_request(self.ctx, parameters)
+        json.dump(request, self)
```

Comment:
Tornado handlers must either use `self.write(...)` (sync mode --> here) or `self.finish(...)` (sync mode), otherwise the  handler can not finish thus data may not be fully submitted.